### ISSUE TITLE
Missing labels / accessible names for input and select elements

### ISSUE THREAD

**Issue by aviannamiller** [omeka/plugin-RedactElements#1]
The following are missing labels / accessible names:

- `.new-element-select` select
- checkbox inputs
